Packs/RecordedFuture/Integrations/RecordedFuture/RecordedFuture.yml

```diff
@@ -5,2169 +5,2819 @@ name: Recorded Future v2
 display: Recorded Future v2
```

Comment:
Shouldn't we change the name here as well?

Reply:
No, we want to have Pack name as Recorded Future Intelligence, but we want to leave Integration name as Recorded Future v2 (due to back compatibility, etc)

Pack name vs Integration name